docs/spec/v1beta1/gitrepositories.md

```diff
@@ -207,9 +207,9 @@ To be able to support Azure DevOps a compromise solution was built, giving the u
 option to select the git library while accepting the drawbacks.
 
 | Git Implementation | Shallow Clones | Git Submodules | V2 Protocol Support |
-|---|---|---|---|
-| 'go-git' | true | true | false |
-| 'libgit2' | false | false | true |
+| ---                | ---            | ---            | ---                 |
+| 'go-git'           | true           | true           | false               |
+| 'libgit2'          | false          | false          | true                |
```

Comment:
Unrelated change. I used https://github.com/godlygeek/tabular to automatically format the table.
I can remove it and keep is simple if it'd be easier to maintain.